pkg/api/providers/google/api.go

```diff
@@ -0,0 +1,22 @@
+package google
+
+import (
+	"net/http"
+
+	"glide/pkg/api/providers"
+)
+
+
+func GoogleApiConfig(APIKey string) providers.ProviderApiConfig {
+	return providers.ProviderApiConfig{
+		BaseURL: "https://generativelanguage.googleapis.com/v1beta/models/gemini-pro",
+		Headers: func(APIKey string) http.Header {
+			headers := make(http.Header)
+			headers.Set("Content-Type", "application/json")
```

Comment:
Are not LLM providers all going to be REST/JSON based? 🤔

Reply:
As in we don't need the content-type header?